static/js/place/parent_breadcrumbs.tsx

```diff
@@ -31,8 +33,18 @@ class ParentPlace extends React.Component<ParentPlacePropsType> {
   render(): JSX.Element {
     const num = this.props.parentPlaces.length;
     return (
-      <React.Fragment>
-        <span>A {displayNameForPlaceType(this.props.placeType)} in </span>
+      // TODO(datcom): Please see the extracted output and required compiled input for place_breadcrumb.
+      // We may need to do still fancier things to have the parentPlace(s) be included. That would mean
+      // creating a more complex FormattedMessage type to account for the variable hrefs.
+      <RawIntlProvider value={intl}>
```

Comment:
Is there a reason to use RawIntlProvider over IntlProvider other than syntactic differences? I didn't look too closely at the docs